modules/oadp-operator-supported.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-intro.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-operator-supported_{context}"]
+= {oadp-full} support
+
+.Supported versions of {oadp-short}
+[width="100%",cols="20%,20%,20%,20%,20%, options="header",]
+|===
+
+|Version
```

Comment:
The context with the change from ( n-2 support ) to a date is the EUS for OCP even releases.
We will need to support OCP EUS releases with a viable OADP through the term 2 support cycle so customers can easily transition from EUS to EUS version.  OADP is also a dep for ACM which needs to adhere to EUS support cycles.   Currently ACM releases we care about are all on OADP-1.4.

Reply:
Cool, let us add this as new columns and a link to the EUS & EUS Term 2 doc

![image](https://github.com/user-attachments/assets/b6870270-90f7-4d64-8374-4779fb746778)

i would have added footnotes but the OCP team have an issue with footnotes

Reply:
1.4 EUS term 2 is 27 Jun 2027
1.3 EUS term 2 is 31 Oct 2026

Reply:
https://access.redhat.com/support/policy/updates/openshift_operators

Reply:
@weshayutin - thanks updated